forte/data/ontology/top.py

```diff
@@ -1105,9 +1190,10 @@ def corners(self):
         Get corners of box.
         """
         return [
-            (self._cy + h_offset, self._cx + w_offset)
-            for h_offset in [-0.5 * self._height, 0.5 * self._height]
-            for w_offset in [-0.5 * self._width, 0.5 * self._width]
+            (self.y0, self.x0),
+            (self.y0, self.x1),
+            (self.y1, self.x0),
+            (self.y1, self.x1),
         ]
 
     @property
```

Comment:
Is `box_min_x` just an equivalent of `max(self.x0, 0)`? Could you explain a little bit on why we need `box_min_x/y` and `box_max_x/y`? I think maybe we can compute IOU or check overlapping directly through `self.x0/1` and `self.y0/1`?